include/matx/core/utils.h

```diff
@@ -146,5 +146,36 @@ __MATX_HOST__ __MATX_DEVICE__ __MATX_INLINE__ auto madd( const T1 &x, const T2 &
   }
 }
 
+template <int RANK, int D>
+auto __MATX_INLINE__ getPermuteDims( const int (&dims)[D] ) {
```

Comment:
Function case isn't matching the other code. Maybe `GetPermuteDims`?

Reply:
This is just a movement of what was already in the code.  We should decide on the actual naming convention we should use.  But that might be outside of the scope of this bug.